def output_notes():
    pass


def error():
    print('Ошибка ввода. Попробуйте еще раз!')
